<launch>
	<arg name="rgb_video" default="True" />
	<arg name="rgb_preview" default="True" />
	<arg name="left" default="True" />
	<arg name="right" default="True" />
	<arg name="disparity" default="True" />
	<arg name="depth" default="True" />
	<node name="depthai_publish_image_stream" pkg="cv" type="depthai_publish_image_stream.py" output="screen">
		<param name="rgb_video" value="$(arg rgb_video)" />
		<param name="rgb_preview" value="$(arg rgb_preview)" />
		<param name="left" value="$(arg left)" />
		<param name="right" value="$(arg right)" />
		<param name="disparity" value="$(arg disparity)" />
		<param name="depth" value="$(arg depth)" />
	</node>
</launch>